zkevm-circuits/src/evm_circuit/execution/precompiles/ec_mul.rs

```diff
@@ -130,23 +175,22 @@ impl<F: Field> ExecutionGadget<F> for EcMulGadget<F> {
         );
 
         cb.condition(
-            not::expr(or::expr([p_is_zero.expr(), s_is_zero.expr()])),
+            // skip ECC table lookup if:
+            // - P == (0, 0)
+            // - s == Fr::zero()
+            // - s == Fr::MODULUS - 1
```

Comment:
do we also need to skip the case where `s == 1`?

Reply:
There is no issue for ECC Circuit to process `s == 1`. I am adding that case (to be generated at random) in the ECC circuit tests